_includes/metas-ogp.html

```diff
@@ -0,0 +1,18 @@
+
+    <title>{% if page.title %}{{ page.title }} | {{ site.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <!-- # Opengraph protocol properties: https://ogp.me/ -->
+    <meta name="author" content="{% if page.author %}{{ page.author | join: ', ' }}{% else %}The {{site.title}} website team, {{ autharray |uniq | join: ", " }}{% endif %}" />
+    {% if page.author %}<meta property="og:type" content="article" />{% endif %}
+    <meta name="twitter:card" content="summary">
+    <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+    <meta name="keywords" content="{% if page.categories %}{{ page.categories | join: ', ' }}{% else %}{% for category in site.categories %}{{ category | first | slugify }}, {% endfor %}{% endif %}" />
+    <meta property="og:title" content="{% if page.title %}{{ page.title }} | {{ site.title }}{% else %}{{ site.title }}{% endif %}"/>
```

Comment:
Any particular reason we shouldn't use the same title logic from the default layout?

```
{{ page.doctitle | default: "Rook" | escape }}
```

Reply:
In this way, site.title is taken from the configuration so the meta-ogp include is more portable across sites (so it's easier also to get improvements to it when needed)

Reply:
Moved out of layout and updated 'Rook' in _config so that it works the same way